### PR TITLE
TV page clock update

### PR DESCRIPTION
**Pull request by davidhtien**
start using time.is

see: https://time.is/widgets

### CODE REVIEW

**Comment by davidhtien** [comment]
clock.js is pretty unreadable :/ unfortunately but it does work, I would refactor but it might not be worth it, also this adds a dependency on time.is which is another thing to consider, I don't think they'll be going anywhere soon but you never know

File: ap/room_reservations/templates/room_reservations/tv_page.html [hunk 1]
```diff
@@ -7,254 +7,232 @@
   <link rel="stylesheet" href="{% static 'room_reservations/css/font.css' %}" />
   <link rel="stylesheet" href="{% static 'room_reservations/css/weather-icons.min.css' %}" />
   <script src="{% static 'room_reservations/js/weather.js' %}"></script>
-</head>
```
Comment (davidhtien) [inline ap/room_reservations/templates/room_reservations/tv_page.html]: I just did some reformatting here, placing everything in the head tag and indenting

File: ap/room_reservations/templates/room_reservations/tv_page.html [hunk 2]
```diff
@@ -7,254 +7,232 @@
...
+    hours = hours % 12;
+    hours = hours ? hours : 12;
+    minutes = minutes < 10 ? '0' + minutes : minutes;
+    var strTime = hours + ':' + minutes + ' ' + ampm;
+    return strTime;
   }
-  $('#daft').text(forecasts[2].day);
-}
 
-function updateClock()
```
Comment (davidhtien) [inline ap/room_reservations/templates/room_reservations/tv_page.html]: I deleted this function

File: ap/room_reservations/templates/room_reservations/tv_page.html [hunk 3]
```diff
@@ -266,12 +244,7 @@
   <div class="row">
 
     <div class="col-sm-7 text-center">
-      <div id="clock-container" class="row">
-        <div class="col-sm-12"><span id="clock"></span><span id="ampm"></span></div>
-      </div>
-      <div id="date-container" class="row">
-        <div class="col-sm-12"><span id="date"></span></div>
-      </div>
+      <span id="Anaheim_z14e"></span>
```
Comment (davidhtien) [inline ap/room_reservations/templates/room_reservations/tv_page.html]: this is where clock.js inserts

File: ap/room_reservations/static/room_reservations/js/clock.js [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+                }
+                if (W == 0) {
+                    W = 52;
+                    if (new Date(y - 1,0,1).getDay() == 4 || new Date(y - 1,11,31).getDay() == 4) {
+                        W = 53;
+                    }
+                }
+                if (p["W"] == "hy") {
+                    if (W == 1) {
+                        pw = "1-ին շաբաթ";
```
Comment (iamsitting) [inline ap/room_reservations/static/room_reservations/js/clock.js]: What is this?

Comment (davidhtien) [reply ap/room_reservations/static/room_reservations/js/clock.js]: bruh I wish I knew

Comment (davidhtien) [reply ap/room_reservations/static/room_reservations/js/clock.js]: ok from a little googling it seems like this has to do with the Armenian calendar... I'll take it out I don't know why it was included in their widget script

**Comment by davidhtien** [comment]
Alright @iamsitting can you take another look? I want to merge it in and test it out to see if it's accurate over multiple days, we can check in again on saturday during service time

**Comment by davidhtien** [comment]
Actually let's try #1273 first, don't merge this yet

**Pull request closed without merging**